[2024-06-18T11:05:29.816+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-18T11:05:29.837+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_dag.second_task scheduled__2023-09-11T00:00:00+00:00 [queued]>
[2024-06-18T11:05:29.846+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_dag.second_task scheduled__2023-09-11T00:00:00+00:00 [queued]>
[2024-06-18T11:05:29.847+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-06-18T11:05:29.864+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): second_task> on 2023-09-11 00:00:00+00:00
[2024-06-18T11:05:29.872+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=4131) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-18T11:05:29.874+0000] {standard_task_runner.py:63} INFO - Started process 4144 to run task
[2024-06-18T11:05:29.873+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'test_dag', 'second_task', 'scheduled__2023-09-11T00:00:00+00:00', '--job-id', '616', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmp_c6_cph6']
[2024-06-18T11:05:29.875+0000] {standard_task_runner.py:91} INFO - Job 616: Subtask second_task
[2024-06-18T11:05:29.922+0000] {task_command.py:426} INFO - Running <TaskInstance: test_dag.second_task scheduled__2023-09-11T00:00:00+00:00 [running]> on host 195daacf7793
[2024-06-18T11:05:30.035+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_dag' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2023-09-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-11T00:00:00+00:00'
[2024-06-18T11:05:30.036+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-18T11:05:30.055+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-18T11:05:30.056+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2024-06-18T11:05:30.068+0000] {subprocess.py:86} INFO - Output:
[2024-06-18T11:05:30.070+0000] {subprocess.py:93} INFO - hey, I am task2 and will be running after task1!
[2024-06-18T11:05:30.071+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-18T11:05:30.072+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-18T11:05:30.101+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=test_dag, task_id=second_task, run_id=scheduled__2023-09-11T00:00:00+00:00, execution_date=20230911T000000, start_date=20240618T110529, end_date=20240618T110530
[2024-06-18T11:05:30.128+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-18T11:05:30.147+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-18T11:05:30.149+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
